server/models/wave.py

```diff
@@ -1,18 +1,20 @@
-from sqlalchemy import Column, Float, Integer, String
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
 
 from server.db.base_class import Base
 
 
-class Historical_Wave(Base):
-    __tablename__ = "Historical Wave Data"
+class Wave(Base):
```

Comment:
Import this class in relative [`__init__.py` file](https://github.com/clairBuoyant/server/blob/main/server/models/__init__.py) for convenience.

And also in [this file](https://github.com/clairBuoyant/server/blob/main/server/db/base.py) to support working with `alembic` for migrations.

Reply:
Not really sure what I'm going to be importing here? Those files don't have any classes, they just have import statements...

Reply:
You're just continuing that pattern by appending the new classes like so:

```python

from server.models.wave import WaveDatum  # noqa: F401
```

This is similar to an "index.js" file.